Jenkinsfile

```diff
@@ -88,12 +88,17 @@ find test-results
 
 def buildImage(params, String suffix) {
     stage("Building disk images") {
+	autobootArgs=""
+	if (suffix.contains("morello") && GlobalVars.selectedPurecapKernelArchitectures.contains(suffix)) {
+	    // If we are building the morello purecap kernel, we need this to be able to boot it for tests
+	    autobootArgs="--disk-image/no-autoboot"
```

Comment:
This raises a question: what is the intended purpose of the disk images we archive? Is it for use in other Jenkins jobs where you want to be able to specify the kernel, or is it for downloading and using as a friendly pre-built image with autoboot?

Reply:
Good point, I guess we want to archive an user-friendly image... We _could_ delete `/boot/loader.conf` in the image we archive instead of generating a new image but it feels hacky

Reply:
How reliable is just interrupting the autoboot countdown? We've got a whole 10s to send a single keystroke...

Reply:
Removing this as autoboot should now be supported by cheribuild